src/components/combat/Combat.js

```diff
@@ -0,0 +1,53 @@
+import combatStyle from './Combat.module.scss'
+import React from 'react'
+// import Button from '../button'
+import IzzyButton from 'components/izzyButton/'
+import { BUTTON_TYPE } from 'components/izzyButton/izzyButton'
```

Comment:
We should import and export it from `index.js`, we won't need to write `/ComponentName`, only the folder name is the perfect scheme.

```suggestion
import { BUTTON_TYPE } from 'components/izzyButton'
```

Same for your components, you must create `index.js` in your component folder.

> index.js
```
import Combat from './Combat

export default Combat
```

You can import it as `components/combat` anywhere. Feel free to ask if you have any questions. Thank you!